edx_event_bus_kafka/internal/consumer.py

```diff
@@ -106,9 +103,7 @@ def consume_indefinitely(self):
             full_topic = get_full_topic(self.topic)
             self.consumer.subscribe([full_topic])
 
-            # TODO (EventBus):
-            # 1. Is there an elegant way to exit the loop?
-            # 2. Determine if there are other errors that shouldn't kill the entire loop
+            # TODO: Handle exceptions at all https://github.com/openedx/event-bus-kafka/issues/62
```

Comment:
What does this comment mean? Is it meant to be a question (see possible suggestion)? Other?
```
TODO: Handle exceptions at all? See https://...
```

Reply:
What I meant was "we're not handling exceptions at all (but we should)" -- too terse, though. Will reword!